target/scripts/start-mailserver.sh

```diff
@@ -174,27 +182,18 @@ function _register_functions() {
 # ? >> Executing all stacks / actual start of DMS
 # ------------------------------------------------------------
 
-# Ensure DMS only adjusts config files for a new container.
-# Container restarts should skip as they retain the modified config.
-if [[ ! -f /CONTAINER_START ]]; then
-  _early_supervisor_setup
-  _early_variables_setup
+_early_supervisor_setup
+_early_variables_setup
 
-  _log 'info' "Welcome to docker-mailserver ${DMS_RELEASE}"
+[[ -f /CONTAINER_START ]] && _log 'info' 'Container was restarted. Skipping most setup routines.'
 
-  _register_functions
-  _check
-  _setup
-  _run_user_patches
-else
-  # container was restarted
-  _early_variables_setup
+_log 'info' "Welcome to docker-mailserver ${DMS_RELEASE}"
 
-  _log 'info' 'Container was restarted. Skipping setup routines.'
-  _log 'info' "Welcome to docker-mailserver ${DMS_RELEASE}"
+_register_functions
+_check
+_setup
```

Comment:
`_setup` is called, even when the container was restarted?

I would revert the 2nd commit. Using one `if` clause at the bottom instead of three, makes it easier to see, what's exactly done on a fresh container / restarted container.

Reply:
Agreed, looks like we can just have a single conditional that calls `_setup` only, we just need to look at two of the setup registered functions being moved to the `_check` function group instead as they're more generic to sanitization/prep before starting services.